netaddr.go

```diff
@@ -846,3 +871,422 @@ func (p *IPPrefix) UnmarshalText(text []byte) error {
 func (p IPPrefix) String() string {
 	return fmt.Sprintf("%s/%d", p.IP, p.Bits)
 }
+
+// LastIP returns the last IP in the prefix.
+func (p IPPrefix) LastIP() IP {
```

Comment:
Bit weird to have this exported by itself. Maybe pair with FirstIP if you want this to be a public API? Or unexport.

Reply:
Oh, I meant to unexport this once I added the Range method.